app/views/users/profile.html.erb

```diff
@@ -7,8 +7,24 @@
   <%= render :partial => "map/userLeaflet" , locals: { haslocation: false, user: @profile_user } %>
   <% end %>
 
-  <hr />
-  <br />
+  <br>
```

Comment:
please keep up with the coding style, br should have a closing tag as it earlier had.